Fixed #32749 -- Doc'd PyMemcacheCache defaults. (#15796)
django/django: docs/topics/cache.txt

        }
    }

> ``pymemcache`` by default sets the following options (you can override them on
> your :setting:`OPTIONS <CACHES-OPTIONS>`)::

Reviewer: The `PyMemcacheCache` backend sets this options, not `pymemcache` itself, so maybe:
```suggestion
By default, the ``PyMemcacheCache`` backend sets the following options (you can
override them in your :setting:`OPTIONS <CACHES-OPTIONS>`)::
```

    'OPTIONS': {
        'allow_unicode_keys': True,
        'default_noreply': False,
        'serde': pymemcache.serde.pickle_serde,

>     }

Reviewer: I would move this to the ["Memcached" section](https://docs.djangoproject.com/en/dev/topics/cache/#memcached), just before _"A final point about...."_:
```
By default, the ``PyMemcacheCache`` backend sets the following options (you can
override them in your :setting:`OPTIONS <CACHES-OPTIONS>`)::

    'OPTIONS': {
        'allow_unicode_keys': True,
        'default_noreply': False,
        'serde': pymemcache.serde.pickle_serde,
    }

A final point about...
```

Author: Yes, you're right, it fits better there.

        }
    }

``pymemcache`` by default sets the following options (you can override them on
your :setting:`OPTIONS <CACHES-OPTIONS>`)::

    'OPTIONS': {
        'allow_unicode_keys': True,
        # WARNING: changing default_noreply to True is not supported.

>         # See https://code.djangoproject.com/ticket/32749#comment:3

Reviewer: Changing `default_noreply` breaks our test suite, but folks in general can do this. I would drop this comment.
